### DISCUSSION TITLE
在已连接US状态下有些网址访问不了，反而SG可以，使用困难，请求退款。

### DISCUSSION THREAD

**Discussion by lizihui12345678** [getlantern/lantern-forum-cn#772] [Q&A]
绑定邮箱：1091791071@qq.com
支付截图：
![微信图片_20240810170844](https://github.com/user-attachments/assets/3a6232f6-6cad-4ae9-8b16-f664b1445d6a)

问题：在已连接US状态下有些网址访问不了，反而SG可以，使用困难，请求退款。

无法访问的网址比如 adobe stock：
![image](https://github.com/user-attachments/assets/d0e2bbfb-171c-4e97-bfc5-3767a595e84c)
![image](https://github.com/user-attachments/assets/a53ef5f3-992d-416c-8417-d67930f4af5c)
![image](https://github.com/user-attachments/assets/6258c7aa-e568-4d87-87db-e858ab0f1195)

难用，且自己无法手动更换线路，请求退款，感谢。

**Comment by MaxMKMax** [comment]
@lizihui12345678 您好. 谢谢您支持蓝灯. 明白了我会帮您提交退款!

**Comment by MaxMKMax** [comment]
@lizihui12345678 在提交退款申请之前可以帮我个忙吗? 帮我们提交应用内报告一下. 我们也想查看问题.

您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by lizihui12345678** [reply]
已提交日志，收到码：75641188

**Reply by lizihui12345678** [reply]
有四天了，还没有收到退款哦，麻烦及时处理，感谢

**Comment by lizihui12345678** [comment]
已提交日志，收到码：75641188

Mike K ***@***.***> 于2024年8月11日周日 21:51写道：

**Comment by lizihui12345678** [comment]
@Mike777Khoo 已提交日志，收到码：75641188 
还未收到退款，麻烦有空及时处理， 感谢
